### PR TITLE
add function to remove row on click

### PR DESCRIPTION
**Pull request by brobuck87**
Trying to get the row to add a button to allow it to delete the current row

### CODE REVIEW

**Review by johnsickels — CHANGES_REQUESTED** [review]
Logic looks good but needs formatting

`option + shift + f` or right click and format document

File: assets/js/script.js [hunk 1]
```diff
@@ -30,12 +30,22 @@ function handleFormSubmit(event) {
      //put in tr
      tableRow.append(tableData);
      };
-  
+     var buttonTD = $('<td>')
+     var deleteButton = $('<button>');
+     deleteButton.attr('type', 'button') 
+     deleteButton.attr('class', 'btn btn-danger delete fa fa-trash')
```
Comment (johnsickels) [inline assets/js/script.js]: Love the font awesome here

File: assets/js/script.js [hunk 2]
```diff
@@ -30,12 +30,22 @@ function handleFormSubmit(event) {
...
      };
-  
+     var buttonTD = $('<td>')
+     var deleteButton = $('<button>');
+     deleteButton.attr('type', 'button') 
+     deleteButton.attr('class', 'btn btn-danger delete fa fa-trash') 
+     buttonTD.append(deleteButton)
+     tableRow.append(buttonTD)
+  $('#project-info tbody').on('click', '.delete',function(){
+    $(this).closest('tr').remove();
```
Comment (johnsickels) [inline assets/js/script.js]: Nice use of jQuery methods

File: assets/js/script.js (hunk 1 above)
Comment (brobuck87) [reply assets/js/script.js]: Yeah, thanks google! I'd used them in the past

File: index.html [hunk 3]
```diff
@@ -55,6 +58,7 @@ <h5 class="card-title">How to use this project tracker</h5>
             <th scope="col">Due Date</th>
             <th scope="col">Days Until Due</th>
             <th scope="col">Total Earned Estimate</th>
+            <th scope="col">Delete Project</th>
```
Comment (johnsickels) [inline index.html]: You don't really need the column header if you don't want it

**Review by johnsickels — APPROVED** [review]
LGTM!

**Pull request merged**